scheduler/scheduler_dfs.go

```diff
@@ -47,69 +46,50 @@ func (p *Plugin) syncDfs(ctx context.Context, spec specs.Source, client schema.C
 					scope.SetTag("table", table.Name)
 					sentry.CurrentHub().CaptureMessage("duplicate client ID in " + table.Name)
 				})
-				p.logger.Warn().Str("client", c.ID()).Str("table", table.Name).Msg("multiplex returned duplicate client")
+				s.logger.Warn().Str("client", c.ID()).Str("table", table.Name).Msg("multiplex returned duplicate client")
 			}
 		}
 		preInitialisedClients[i] = clients
 		// we do this here to avoid locks so we initial the metrics structure once in the main goroutines
 		// and then we can just read from it in the other goroutines concurrently given we are not writing to it.
-		p.metrics.initWithClients(table, clients)
+		s.metrics.initWithClients(table, clients)
 	}
 
-	// We start a goroutine that logs the metrics periodically.
-	// It needs its own waitgroup
-	var logWg sync.WaitGroup
-	logWg.Add(1)
-
-	logCtx, logCancel := context.WithCancel(ctx)
-	go p.periodicMetricLogger(logCtx, &logWg)
```

Comment:
We should probably open an issue to bring back this functionality

Reply:
We shouldn't have this functionality anymore, as we will move towards telemetry for metrics